src/node_file.cc

```diff
@@ -854,6 +873,45 @@ void FromNamespacedPath(std::string* path) {
 #endif
 }
 
+
+static inline int GetValidMode(Environment* env,
+    Local<Value> mode_v, std::string type) {
+  if (!mode_v->IsInt32() && !mode_v->IsNullOrUndefined()) {
+    env->isolate()->ThrowException(ERR_INVALID_ARG_TYPE(
```

Comment:
You can use THROW_ERR_INVALID...

Reply:
Oh, nice. I was confused if it would be the same thing. I will refactor.